<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CanteenManagerWindow</class>
 <widget class="QMainWindow" name="CanteenManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1231</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="myCanteenTableWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>671</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>431</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ÊàëÁÆ°ÁêÜÁöÑÈ£üÂ†Çüçö</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addCanteenButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>620</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ê∑ªÂä†</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteCanteenButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>620</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Âà†Èô§</string>
    </property>
   </widget>
   <widget class="QPushButton" name="canteenRefreshButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Âà∑Êñ∞</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1231</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
